mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -53,6 +53,8 @@ private[shared] object SharedParamsCodeGen {
       ParamDesc[Int]("checkpointInterval", "checkpoint interval (>= 1)",
         isValid = "ParamValidators.gtEq(1)"),
       ParamDesc[Boolean]("fitIntercept", "whether to fit an intercept term", Some("true")),
+      ParamDesc[String]("handleInvalid", "how to handle invalid entries",
```

Comment:
Please make docs more explicit.  It's very helpful to users.  Here, it'd be nice to state the options and what they mean: "Options: skip (filter out rows with bad values) or error (throw an error).  Note that skipping rows means the Transformer can output a smaller dataset."